cmd/loop/loopout.go

```diff
@@ -41,7 +41,7 @@ var loopOutCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name:  "amt",
-			Usage: "the amount in satoshis to loop out",
+			Usage: "run the command loop terms to get the minimum & maximum amount on the server. ",
```

Comment:
> run the command loop terms to get the minimum & maximum amount on the server.

better rephrase as "the amount in satoshis to loop out, consult the loop terms command to get the minimum & maximum loop amounts supported by the server."

Reply:
updated it to this 

https://github.com/lightninglabs/loop/pull/543/commits/155ccd5ce6f72ec9085dce4742df988602f7cdc9

Reply:
I think its in one commit now?